Chapters/2-0-Part2.md

```diff
@@ -0,0 +1,27 @@
+# Part II: Kipnu
+
+_None shall terminate the flow of consciousness_
+
+&nbsp;
+
+_whether their own or another's;_
+
+_whether temporarily or permanently;_
+
+_whether directly or indirectly;_
+
+_whether by force or by accident;_
```

Comment:
```suggestion
_whether with touch or with induction;_

_whether by intent or by accident;_
```

idk. I think you can use better words, not sure if the ones I suggested get your intentions across. Maybe we can workshop this sometime.

Reply:
I think  got some better ones in